backend/spec/requests/api/v1/investors_spec.rb

```diff
@@ -126,6 +128,30 @@
 
           run_test!
         end
+
+        context "account language" do
+          let(:investor_in_pt) {
+            account = @investor.account
+            account.update_columns(about_en: "About EN", about_es: "About ES", about_pt: "About PT", language: "pt")
```

Comment:
To be honest, something about these `let` examples where you update columns feels wrong. I thing it makes more sense to just create new record which suits your needs or put this data manipulation into `before` block.

Reply:
yes! it feels so wrong, but I got so tired of trying to create fresh objects, I get validation errors about duplicated email which I think come from a user object created somewhere I cannot put a finger on 😫

Reply:
right, it seems to work in `before_all` 👍